### DISCUSSION TITLE
Hydrogen and Deuterium provide same quantum chemical infoamtion

### DISCUSSION THREAD

**Discussion by teltim** [rdkit/rdkit#6460] [Q&A] [answered]
As my interest, I cheked quantum chemical information of Hydrogen and Deuterium with psi4 framwork.

.sdf structure files are downloaded here.
Hydrogen https://pubchem.ncbi.nlm.nih.gov/compound/Hydrogen
Deuterium https://pubchem.ncbi.nlm.nih.gov/compound/Deuterium  

Moleclue  : Volume	   Energy	       HOMO	LUMO	Atom_num	Mass	Density
Hydrogen : 10.136	-1.145880789	-0.596845574	0.169278454	2	2.015650      0.198860504
Deuterium : 10.136	-1.145880775	-0.59686526	0.169288162	2	2.015650      0.198860504

Both are almost same, Is this reasonable? I thought that Hydrogen and Deuterium should be different each other, for instance, Mass?

**Comment by greglandrum** [comment, marked as answer]
I don't think this is actually an RDKit question.
